core/store/migrate/migrations/0083_add_keeper_confirmations_to_spec.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+ALTER TABLE keeper_specs
+    ADD COLUMN confirmations integer;
```

Comment:
Do we want to call it `min_required_incoming_confirmations` just to keep it consistent?

Reply:
I used the same name as in the VRF spec, should we change that one too?

Reply:
Yes, to be honest. "confirmations" is ambiguous